### ISSUE TITLE
SSR render incomplete

### ISSUE THREAD

**Issue by happyleavesaoc** [vkurko/calendar#205]
I'm attempting to use `event-calendar` with sveltekit's SSR-only mode, but the server-rendered calendar is not fully initialized. I suspect that `onMount` may be the culprit here.

Example component:

```js
<script>
import Calendar from '@event-calendar/core';
import DayGrid from '@event-calendar/day-grid';
import '@event-calendar/core/index.css';

let options = {
    view: 'dayGridMonth',
    events: [
        {start: '2023-11-01', end: '2023-11-02', title: 'event', allDay: true}
    ]
};

</script>

<Calendar options={options} plugins={[DayGrid]} />
```

Actual output:
![image](https://github.com/vkurko/calendar/assets/11217032/b59fa7fd-0e3b-4d1b-bcf2-5f82f9178c99)

Expected output:
![image](https://github.com/vkurko/calendar/assets/11217032/bb6b37ed-7d71-42ce-a33d-957821bf4b80)

(this was generated by turning on CSR in addition to SSR)

**Comment by vkurko** [comment]
Yes, unfortunately, that's the way it works now. The event placement algorithm at some points requires information about the size of DOM elements, which becomes available only when rendering these elements in the browser. Also, various optimizations use deferred computation, which is not performed in SSR.

I'll leave this ticket open so that I can improve SSR support over time (if possible).

**Comment by vkurko** [comment]
Just for reference https://github.com/vkurko/calendar/discussions/162#discussioncomment-6572376
